sdv/constraints/tabular.py

```diff
@@ -68,6 +69,11 @@ def _run(self, function, table_data, reverse=False):
         return table_data
 
     def _run_transform(self, table_data):
+        if self._columns:
+            missing_columns = [col for col in self._columns if col not in table_data.columns]
```

Comment:
minor: since we're not using any of the columns we could do 
```
if any(col not in table_data.columns for col in self._columns)
```
so that it terminates early.

Reply:
Sounds good, made the change